code/src/App.js

```diff
@@ -1,9 +1,103 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Radiobuttons } from './Components/Radiobuttons'
+import { TextInput } from './Components/TextInput'
+import { Dropdown } from './Components/Dropdown'
+import { Summary } from './Components/Summary'
+import { Intro } from './Components/Intro'
 
 export const App = () => {
+
+  const [name, setName] = useState('')
+  const [phoneNumber, setPhoneNumber] = useState('')
+  const [email, setEmail] = useState('')
+  const [interest, setInterest] = useState('')
+  const [option, setOption] = useState('')
+  const [section, setSection] = useState('Intro')
+
+  const handleSubmit = event => {
+    event.preventDefault()
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <> 
```

Comment:
No need to use React Fragment if you return only one element (`form`) from your component